src/Symfony/Component/Messenger/Transport/Doctrine/DoctrineReceiver.php

```diff
@@ -46,6 +47,10 @@ public function get(): iterable
     {
         try {
             $doctrineEnvelope = $this->connection->get();
+        } catch (RetryableException $exception) {
+            // Do nothing when RetryableException occurs.
```

Comment:
please add "as it will likely be resolved on the next call to get()".

Reply:
updated